content/filters/index.md

```diff
@@ -0,0 +1,47 @@
+# Filters
+
+Filters are simple methods that modify the output of numbers, strings, variables and objects. They are placed within an output tag `{{ }}` and are separated with a pipe character `|`.
+
+#### Input
+
+~~~ liquid
+  <!-- rental.title = 'Great Boat' -->
+    {{ rental.title | upcase }}
+~~~
+
+#### Output
+
+~~~
+  GREAT BOAT
+~~~
+
+In the example above, `rental` is the object, `title` is its attribute, and `upcase` is the filter being applied.
```

Comment:
I think using `the` is correct here.
Its the second usage, as we are continuing context from:
`Filters are simple methods that modify the output of numbers, strings, variables and objects.`